client/signup/main.jsx

```diff
@@ -428,6 +429,7 @@ class Signup extends React.Component {
 			isNewUser,
 			hasCartItems,
 			isNew7DUserSite,
+			isBlankCanvas: isBlankCanvasDesign( dependencies.selectedDesign ),
```

Comment:
Shouldn't we do the check like in `client/landing/gutenboarding/hooks/use-on-site-creation.ts`? Or maybe update `isBlankCanvasDesign` to return `false` when called with `undefined`?
```suggestion
			isBlankCanvas: dependencies.selectedDesign && isBlankCanvasDesign( dependencies.selectedDesign ),
```

Reply:
> Or maybe update isBlankCanvasDesign to return false when called with undefined?

Thanks for the suggestion! I've updated the `isBlankCanvasDesign` util by making the design argument optional in 66decdb.